src/Entities/AuditEntities/UserADT.ts

```diff
@@ -1,28 +1,24 @@
 // TODO: Update modifiedBy column to log user who made changes
 import { AuditingAction, AuditingEntity, AuditingEntityDefaultColumns } from 'typeorm-auditing';
 import { User } from '../User';
-import { ManyToMany, JoinTable, Column } from 'typeorm';
-import { RoleADT } from './RoleADT';
+import { Column, BeforeInsert } from 'typeorm';
+import { appDataSource } from '../../Repositories/data-source';
 
 @AuditingEntity(User, { name: 'adt_user' })
 export class UserADT extends User implements AuditingEntityDefaultColumns {
     readonly _seq!: number;
     readonly _action!: AuditingAction;
     readonly _modifiedAt!: Date;
 
-    @ManyToMany(() => RoleADT, (role) => role.users, { nullable: true })
```

Comment:
Denne her vil jeg nok beholde for at logge hvornår/hvis en user får ændret rolle på sig.